app/controllers/products_controller.rb

```diff
@@ -0,0 +1,49 @@
+class ProductsController < ApplicationController
+	def index 
+		@products = Product.all
+	end
+
+	def show
+		@product = Product.find(params[:id])
+	end
+
+	def new
+		@product = Product.new
+	end
+
+	def edit
+		@product = Product.find(params[:id])
+	end
+	
+	def create
+		@product = Product.new(params.require(:product).permit(:title, :description, :price, :photo))
```

Comment:
Since you already have a method called `product_params`. Use it.
`@product = Product.new(product_params)`.